tests/unit/test_cloudformation.py

```diff
@@ -1,40 +1,8 @@
-import re
```

Comment:
I didn't see much of a value at this point for these resolve unit tests since I'd prefer to use proper integration tests here soon.